### ISSUE TITLE
small BUGS and problems

### ISSUE THREAD

**Issue by tobyplowy** [D00Med/farlands#101]
report your small bugs and problems here

**Comment by tobyplowy** [comment]
@D00Med items dount burn when they hit lava or fire

**Comment by DS-Minetest** [comment]
> items dount burn when they hit lava or fire

That's rather a missing feature than a bug.

**Comment by tobyplowy** [comment]
@DS-Minetest ok yeh but it needs to be fixed @D00Med ping

**Comment by tobyplowy** [comment]
fences and cobblestone walls shuld connect to evryting

fences only connect to wood and cobblestone walls only connect to stone (i hate this it ruins my buildings XD)

**Comment by DS-Minetest** [comment]
@tobyplowy Fences and walls are connected_nodeboxes, they can only connect to specific nodes or node groups.

**Comment by tobyplowy** [comment]
@DS-Minetest is it possible to add more node groups?

**Comment by DS-Minetest** [comment]
@tobyplowy Sure.

**Comment by tobyplowy** [comment]
@DS-Minetest Ok I found it and it works (on wood and stone) but why can't I make it work with dirt?

**Comment by DS-Minetest** [comment]
@tobyplowy You could add all dig groups (cracky, choppy, oddy_diggable_by_hand, …), that would include nearly everything.

**Comment by tobyplowy** [comment]
@DS-Minetest I did that but it doesn't seem to work

**Comment by DS-Minetest** [comment]
@tobyplowy Have you added it here: https://github.com/D00Med/farlands/blob/master/mods/default/functions.lua#L286 ?

**Comment by tobyplowy** [comment]
@DS-Minetest Yes

**Comment by DS-Minetest** [comment]
You are sure, you haven't forgotten a `,` or the `group:` prefix?

**Comment by tobyplowy** [comment]
@DS-Minetest Yes I'm sure I just added stone and that works :/

**Comment by DS-Minetest** [comment]
Hm, I don't know what the problem is. :confused:

**Comment by tobyplowy** [comment]
@DS-Minetest well if you don't know I'm definitely not gonna know it :/

**Comment by D00Med** [comment]
"group:fence", "group:wood", "group:tree", "group:wall", "group:stone"
You should add "group:cracky", "group:crumbly", "group:snappy", "group:fleshy"
Maybe not snappy and fleshy though, since they are used for plants

**Comment by tobyplowy** [comment]
Jungle wood needs a name maybe Rosewood

**Comment by tobyplowy** [comment]
@D00Med can you add them for me?

**Comment by DS-Minetest** [comment]
MTG's jungle wood is rosewood.

**Comment by DS-Minetest** [comment]
MTG's jungle wood is rosewood.
The description could be changed.

**Comment by LoneWolfHT** [comment]
Can't craft willow wood into planks and I can't collect dirty_water_source with a bucket. I tried both Right Click and Left Click on the dirty_water_source with a bucket in hand. I can collect normal water_source though.
The poison textures haven't been updated yet. (It shows green bread when you eat a red mushroom)

**Comment by D00Med** [comment]
@LoneWolfHT Thanks for pointing these out, I'll work on fixing them

**Comment by LoneWolfHT** [comment]
I don't think any of the new (non-default) trees are able to craft into planks.

**Comment by tobyplowy** [comment]
@LoneWolfHT the things you reported should be fixed now

Have any other problems or perhaps a request?

**Comment by tobyplowy** [comment]
> Are the wild oats supposed to randomly drop oats when mined?

I plan that for future content https://github.com/D00Med/farlands/issues/93

**Comment by LoneWolfHT** [comment]
Ok Maybe add water to the game? That would make it a bit more rouge-like if that's what you're aiming for.

**Comment by tobyplowy** [comment]
> water HUD

I'm not a fan of that sorry

> craftguide

That is planed I'm waiting on @GreenXenith for that

**Comment by LoneWolfHT** [comment]
You know what I meant by water HUD then? You have to drink every once and a while so you don't die of thirst?

> <tobyplowy> That is planed I'm waiting on @GreenXenith for that
Ok cool!
 
**Requests**

1. Could you make snow-balls/blocks smeltable into a water bucket/source?

2. Add more hostile mobs? Also, here's something I found that you might like the idea of: https://forum.minetest.net/viewtopic.php?f=9&t=17922

**Comment by LoneWolfHT** [comment]
Are coconuts supposed to be crafted from **palm_leaves (coconut)**? How do I get them?

**Comment by D00Med** [comment]
@LoneWolfHT right click the leaves

**Comment by LoneWolfHT** [comment]
> @LoneWolfHT right click the leaves
Oh ok thanks lol
Is it possible to add a sort of dual wield? My idea: add an extra armor slot for 1 torch only. When in the slot you can walk around with a torch in your left hand and another thing in the right. But decrease mining/fighting speed when doing this. That would be a cool addition.

**Comment by D00Med** [comment]
@LoneWolfHT Dueal wield would be awesome, but I don't think I could do that. 
That's a really good idea. I still need to improve the torch code though, since it doesn't seem to work with multiple players.

**Comment by LoneWolfHT** [comment]
Did you do the torch code yourself or was it from a mod?

**Comment by tobyplowy** [comment]
Dueal wield is a cool feature (definitely something I whant) but it's not possible in the current minetest version @wuzzy2 can't do it so no one can hes the god of subgames XD

Also @LoneWolfHT if you're interested in helping with this subgame be sure to do so the more poeple help the better!

**Comment by LoneWolfHT** [comment]
> Also @LoneWolfHT if you're interested in helping with this subgame be sure to do so the more poeple help the better!

Yes I am interested in helping but not much good at anything but ideas :P
You know how the HUD works? Would it be possible to but a 2d image of a torch where the left hand would be?
Also: Maybe reduce the chances of finding a diamond pick in the mines. I have 3 from one shaft :P
Do you have any ideas for an adventure map? I was thinking I could make one and keep it up-to-date with the latest version of this game. IMO I'm good at story ideas also :P

**Comment by LoneWolfHT** [comment]
Where do I post textures I've made for the game?
Here's my texture for the diamond:
![image](https://user-images.githubusercontent.com/28972859/28046325-b40be33a-6597-11e7-8475-3b5df8ba4307.png)

**Comment by D00Med** [comment]
@LoneWolfHT I coded the wielded torch light myself. A hud image of a torch would work.
Posting them here is fine

**Comment by LoneWolfHT** [comment]
I forgot to ask if I could make an Issue for contributing textures.
I think there's a mod out there that does the same thing as your wielded-torch-light mod too. I'll go search up that link.
Here's something you might like also: https://forum.minetest.net/viewtopic.php?f=11&t=15721&hilit=torch#p237016

**Comment by LoneWolfHT** [comment]
Walking-torch-light (most updated version I think) https://forum.minetest.net/viewtopic.php?f=11&t=14359

**Comment by D00Med** [comment]
@LoneWolfHT Your diamond texture doesn't really fit into the game, @tobyplowy do we use it?
Anyway, you can make a new issue for textures, that'd be really useful.
Yeah I think it might have been Blockmen's torches mod. That looks good too, thanks.
Oh ok, that might mean the code is already in the game.

**Comment by LoneWolfHT** [comment]
What do you want the diamond to look like? I can change np.

**Comment by LoneWolfHT** [comment]
Could you make both chest types nameable? That would really help sorting things lol.
(Like the protected chest in protector_redo)

**Comment by tobyplowy** [comment]
@LoneWolfHT @D00Med eeeeeh yeh that diamond texture looks a bit too simple also it doesn't fit the style I already made a Diamond Texture but I haven't added it to the subgame yet (I still need to mess with the color and contrast so it fits the game)

**Comment by tobyplowy** [comment]
@LoneWolfHT also if you talk about art (textures and images) use @tobyplowy so I get notified
I am also the lead artist of the project so all art needs to run by me first so I can check the colour balance and details :)

**Comment by D00Med** [comment]
@tobyplowy Would it be a good idea to show us your progress on making a texture? if you don't mind of course.

**Comment by tobyplowy** [comment]
@D00Med yes I will do that For the diamond textures

**Comment by D00Med** [comment]
@tobyplowy Thanks :D

**Comment by tobyplowy** [comment]
@D00Med If I remember that is! also shuld I record it?

Because I can't really do it with steps because I just go all over the place there's no way to make easy staps
So i will have to record it but that will be hard because I don't have capable computer at the moment but hopefully next week I will (I will buy a new PC a good one if I have Enough money otherwise I will have to keep saving) and otherwise I will just have to test some things (like low resolution recording like I did for the merchant vid but that may look bad and unuseble)

**Comment by D00Med** [comment]
@tobyplowy Yeah that might be better.

**Comment by D00Med** [comment]
@tobyplowy  good luck getting a new PC

**Comment by LoneWolfHT** [comment]
My full-health tamed chicken is missing from a 3 node high double thick wall of pine fence.

**Comment by tobyplowy** [comment]
@LoneWolfHT that seems like a problem I will tell it to d00med :)

**Comment by D00Med** [comment]
@LoneWolfHT There's not much that can be done about that unfortunately. I could try making the collision box a little bigger

**Comment by tobyplowy** [comment]
@D00Med I think it's because it despawned

**Comment by D00Med** [comment]
@tobyplowy I think tamed animals don't despawn

**Comment by LoneWolfHT** [comment]
I just realized the bug:Me -_- I did a clear objects quick while in my house and I think it cleared the chicken too. Might want to enlarge the collision box anyway though. I've found him in between the holes in a fence that you get when making a double thick wall. Sorry for that false alarm.

**Comment by LoneWolfHT** [comment]
@D00Med @tobyplowy I found a weird bug:
Fish can *swim* on land. I forgot to take a screenshot sadly. If you want I could reproduce it and take a picture.

**Comment by D00Med** [comment]
@LoneWolfHT Ok, that might be a bit hard to fix, but I'll try

**Comment by LoneWolfHT** [comment]
Make them despawn if They aren't in water maybe?

**Comment by D00Med** [comment]
@LoneWolfHT I'm just going to try reducing their stepheight for now, and see what happebs

**Comment by lefty-studios** [comment]
Bamboo crafts unknown Item

**Comment by tobyplowy** [comment]
@lefty-studios I'm working on fixing it :)

**Comment by tobyplowy** [comment]
@lefty-studios it's fixed thanks for reporting

**Comment by LoneWolfHT** [comment]
You should make players lose air when in kelp or other sea-plants. That would solve the air problem.
Also: You don't have any inventory textures for Pink Coral.

**Comment by D00Med** [comment]
@LoneWolfHT I'm thinking of removing them until the visible air bubble thing can be fixed.
